examples/ssl/example_ssl.cpp

```diff
@@ -4,6 +4,7 @@
 int main()
 {
     crow::SimpleApp app;
+    http_connection_flags f;
```

Comment:
is this needed?

Reply:
Apparently not.  :)  I did notice it in the diffs, but put it down to ReSharper's refactoring and assume it served some purpose.  Clearly it does not.  I will make ammends.

On the topic of example code, this section in example_vs.cpp (not sure if `vs` is intended to represent visual studio?) causes an unhelpful error in routing.h due to it returning a void.

```cpp
    app.route_dynamic("/add/<int>/<int>")
    ([](crow::response& res, int a, int b){
        std::ostringstream os;
        os << a+b;
        res.write(os.str());
        res.end();
    });

```

I'm not sure if this works on other compilers, but MSVC quite reasonably complains that

```
Error	C2440	'<function-style-cast>': cannot convert from 'void' to 'crow::response'e:\git\crow\include\crow\routing.h	178	
```

Reply:
This is interesting, it's not meant to return a response if `crow::response&` is an argument. I'll have to look into that.

Reply:
I hadn't notice that detail, though it seem to be a little vague in example_vs with the third example below return a response while taking crow::response& as an argument as well -- unless different rules apply to due to the chained called to `methods()`.

```cpp
    app.route_dynamic("/hello/<int>")
    ([](int count){
        if (count > 100)
            return crow::response(400);
        std::ostringstream os;
        os << count << " bottles of beer!";
        return crow::response(os.str());
    });

    app.route_dynamic("/add/<int>/<int>")
    ([](crow::response& res, int a, int b){
        std::ostringstream os;
        os << a+b;
        res.write(os.str());
        res.end();
    });

    app.route_dynamic("/add_json")
        .methods(crow::HTTPMethod::Post)
    ([](const crow::request& req){
        auto x = crow::json::load(req.body);
        if (!x)
            return crow::response(400);
        auto sum = x["a"].i()+x["b"].i();
        std::ostringstream os;
        os << sum;
        return crow::response{os.str()};
    });
```